Sources/SwiftProtobuf/Google_Protobuf_Any+Registry.swift

```diff
@@ -32,7 +32,7 @@ internal func typeName(fromMessage message: Message) -> String {
   return messageType.protoMessageName
 }
 
-internal let defaultTypePrefix: String = "type.googleapis.com"
+public let defaultAnyTypeURLPrefix: String = "type.googleapis.com"
```

Comment:
There's a part of me that would like to keep this private.  I wonder why default args have to be public.  If we put the raw string value in each place it wouldn't be exposing a symbol.